tests/acceptance/features/bootstrap/TestingAppContext.php

```diff
@@ -436,6 +441,146 @@ public function theResponseShouldHaveTheAppEnabledStatusOfApp() {
 		}
 	}
 
+	/**
+	 * @Given locking is enabled
```

Comment:
We try to use "has been" sentences for actions like this:'
```
Given locking has been enabled
```

The problem is that:
```
Given locking is enabled
Then locking is enabled
```
both look the same to Behat-Gherkin.

In English, the words could be an action or a test. So we try not to use ordinary "is" sentences. For actions we write:
```
Given something has been done
When the user does something
```
and for then tests we write like:
```
Then something should be set to "xyz"
```